NSS-Day-Cohort-42/rare-media-moguls
# Cb post list

#### Changes Made
1. when the user logs in and the url is on the home page ("/") the user should see all posts on the left hand side and all categories on the right
2. posts should show the user's display name, the date of publication, the post title, and the category of the post
3. posts should be sorted from most recent to least recent
4. when the user clicks on the post title they should be redirected to post detail page
5. application views should not have duplicate routes
​
#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout cb-post-list
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
watchgod request_handler.main
```
5. Test app functionality. 
> login to app
> check that posts show the user's display name, the date of publication, the post title, and the category of the post
> posts should be sorted from most recent to least recent
> when the user clicks on the post title they should be redirected to post detail page
> if you feel the need for seed: https://gist.github.com/heymonicakay/9176ae9b672608777813b237cda403be
6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
